Dte.Common/Content/CustomMessageEmail.cs

```diff
@@ -1,429 +1,48 @@
-                ""
-              >
-                <!--[if mso | IE]>      <table role=""presentation"" border=""0"" cellpadding=""0"" cellspacing=""0"">        <tr>          <td style=""vertical-align:top;width:600px;"">      <![endif]-->
-                <div
-                  class=""mj-column-per-100 outlook-group-fix""
-                  style=""
-                    vertical-align: top;
-                    display: inline-block;
-                    direction: ltr;
-                    font-size: 13px;
-                    text-align: left;
-                    width: 100%;
-                  ""
-                >
-                  <table
-                    role=""presentation""
-                    cellpadding=""0""
-                    cellspacing=""0""
-                    style=""
-                      vertical-align: top;
-                      border-collapse: collapse;
-                      mso-table-lspace: 0pt;
-                      mso-table-rspace: 0pt;
-                      padding: 0 10px;
-                    ""
-                    width=""100%""
-                    border=""0""
-                  >
-                    <tbody>
-                      <tr>
-                        <td
-                          style=""
-                            word-wrap: break-word;
-                            font-size: 0px;
-                            padding: 0px 0px 0px 0px;
-                            border-collapse: collapse;
-                            mso-table-lspace: 0pt;
-                            mso-table-rspace: 0pt;
-                          ""
-                        >
-                          <div
-                            style=""
-                              cursor: auto;
-                              color: #000000;
-                              font-family: Lato, Helvetica, Arial, sans-serif;
-                              font-size: 11px;
-                              line-height: 1.5;
-                              margin: 0 18px;
-                            ""
-                          >
-                            ###BODY_REPLACE###
-                          </div>
-                        </td>
-                      </tr>
-                    </tbody>
-                  </table>
-                </div>
-                <!--[if mso | IE]>      
-                </td></tr></table>      
-                <![endif]-->
-              </td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-      <!--[if mso | IE]>      
-      </td></tr></table>      
-      <![endif]-->
-    </div>
-  </body>
-</html>";
+            return template.Replace(BodyPlaceholder, bodyContent);
+        }
+
+        private static string LoadTemplate()
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var resourceName = assembly.GetManifestResourceNames()
+                .FirstOrDefault(str => str.EndsWith(EmailTemplateResourceSuffix));
```

Comment:
`.Single()` make sure it is only finding one resource and giving you the right one.
Will throw an exception if nothing is found.